packages/client/hmi-client/src/workflow/tera-workflow.vue

```diff
@@ -422,6 +431,9 @@ const closeDrilldown = async () => {
 			timeSpent
 		})
 	);
+	router.push({
+		query: {}
+	});
```

Comment:
When I click the browser back-button, the route changes but the opened operator drilldown is still there.